.github/workflows/pull-request-target.yml

```diff
@@ -39,7 +39,7 @@ jobs:
             1. We cannot merge PRs with conflicting files. These must be fixed.
                 
             Our policies make our code better.
-            
+
```

Comment:
Your PR has too many unnecessary files that don’t relate to your issue. You can from the PR by running the commands below from the root directory of the repository

git add -u
git reset HEAD path/to/file
git commit

Please apply these changes to this file.